src/openrct2/paint/Paint.cpp

```diff
@@ -986,6 +979,16 @@ paint_struct* sub_98199C(
     return ps;
 }
 
+paint_struct* sub_98199C(
+    paint_session* session, uint32_t image_id, int8_t x_offset, int8_t y_offset, int16_t bound_box_length_x,
+    int16_t bound_box_length_y, int8_t bound_box_length_z, int16_t z_offset, int16_t bound_box_offset_x,
+    int16_t bound_box_offset_y, int16_t bound_box_offset_z)
+{
+    return sub_98199C(
+        session, image_id, { x_offset, y_offset, z_offset }, { bound_box_length_x, bound_box_length_y, bound_box_length_z },
+        { bound_box_offset_x, bound_box_offset_y, bound_box_offset_z });
+}
+
```

Comment:
Are there other users of this function? I don't get why this is being kept

Reply:
There are still a lot of remaining calls, too many for one PR.